splinterd/Cargo.toml

```diff
@@ -45,16 +45,18 @@ toml = "0.4.8"
 [features]
 default = ["generate-certs"]
 
+stable = ["default"]
+
 experimental = [
+    "health",
```

Comment:
The order should probably be config-builder, config-toml, circuit-read, health.

Reply:
Oops, fixed it so health is in the correct spot.